webapp/channels/src/packages/mattermost-redux/src/selectors/entities/users.ts

```diff
@@ -221,6 +221,16 @@ export const getCurrentUserMentionKeys: (state: GlobalState) => UserMentionKey[]
             keys.push({key: usernameKey});
         }
 
+        // Add the only highlight without notification keyword
+        if (user.notify_props.highlight_keys) {
+            keys = keys.concat(user.notify_props.highlight_keys.split(',').map((key) => {
+                return {
+                    key,
+                    caseSensitive: true,
+                };
+            }));
+        }
```

Comment:
This seems to work to highlight the words but it seems that this `getCurrentUserMentionKeys` is also called to list in the RHS actions when opening the Mentions RHS. I don't think we want to consider these highlights as mentions given they are not being notified about it